formazione/models.py

```diff
@@ -361,6 +365,33 @@ def non_idonei(self):
     def idonei(self):
         return self.partecipazioni_confermate().filter(esito_esame=PartecipazioneCorsoBase.IDONEO)
 
+    def genera_pdf_firme(self):
+        """
+        Genera il fogli firme delle lezioni del corso.
+        """
+
+        iscritti = [x.persona.nome_completo for x in self.partecipazioni_confermate()]
```

Comment:
Puoi passare un elenco di oggetti `Persona` al template? Es.:
```python
iscritti = [partecipazione.persona for partecipazione \
            in self.partecipazioni_confermate()]
```

Scegliere se visualizzare il nome completo in una unica stringa, oppure separatamente (ad esempio, in due colonne di una stessa riga), e' responsabilita' del template, non del modello. Non e' ideale dover modificare il modello per cambiare un dettaglio puramente estetico.

Reply:
ok